bootstrap.php

```diff
@@ -60,6 +60,15 @@
     exit("Class Gdn not found.");
 }
 
+// Set up the dependency injection container.
+$dic = new \Garden\Container\Container();
+Gdn::setContainer($dic);
+
+$dic->setInstance('Garden\Container\Container', $dic)
+    ->rule('Interop\Container\ContainerInterface')
+    ->setAliasOf('Garden\Container\Container')
+;
```

Comment:
Please no. We chain in about 10,000 places in our code base. Don't invent a new coding standard in the PR for 10,0001.